### PR TITLE
Kubevirt-mce-bm-1129

### PR DESCRIPTION
**Pull request by jechen0648**

### CODE REVIEW

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-mce-baremetalds-conformance

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-mce-baremetalds-conformance

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__periodics.yaml [hunk 1]
```diff
@@ -142,6 +142,16 @@ tests:
...
     workflow: hypershift-kubevirt-csi-e2e
+- as: e2e-kubevirt-mce-baremetalds-conformance
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      CNV_PRERELEASE_VERSION: "4.15"
+      HYPERSHIFT_NODE_COUNT: "2"
+      MCE_VERSION: "2.4"
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.13
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__periodics.yaml:153]: ODF_OPERATOR_SUB_CHANNEL: stable-4.14

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-mce-baremetalds-conformance

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-mce-baremetalds-conformance

**Pull request closed without merging**